modules/tinymce/src/plugins/table/main/ts/ui/Helpers.ts

```diff
@@ -148,7 +148,7 @@ const getSharedValues = <T>(data: Array<T>) => {
 // because some of these are crazy complicated
 
 const getAlignment = (formats: string[], formatName: string, editor: Editor, elm: Node) =>
-  Arr.find(formats, (name) => editor.formatter.matchNode(elm, formatName + name)).getOr('');
+  Arr.find(formats, (name) => !!editor.formatter.matchNode(elm, formatName + name)).getOr('');
```

Comment:
```suggestion
  Arr.find(formats, (name) => editor.formatter.matchNode(elm, formatName + name) !== undefined).getOr('');
```
I think this works, but without strict typing turned on it's hard to be sure.

Reply:
Please use `!Type.isUndefined` instead of this.